shacl/register.ttl

```diff
@@ -19,252 +19,641 @@
 reg:DatacatalogShape
     a sh:NodeShape ;
     sh:targetClass schema:DataCatalog ;
-    rdfs:label "Datacatalogus"@nl, "Data catalog"@en ;
-    rdfs:comment "Een datacatalogus bestaat uit een set van datasetbeschrijvingen"@nl, "A data catalog consists of a set of dataset descriptions"@en ;
+    sh:name "Datacatalogus"@nl,
```

Comment:
Why did you change `rdfs:label` to `sh:name`? This is not documented in the PR’s description.